plugins/builddep.py

```diff
@@ -94,9 +94,9 @@ def configure(self):
         # enable source repos only if needed
         if not (self.opts.spec or self.opts.srpm):
             for pkgspec in self.opts.packages:
-                if not (pkgspec.endswith('.src.rpm')
-                        or pkgspec.endswith('nosrc.rpm')
-                        or pkgspec.endswith('.spec')):
+                if not (pkgspec.endswith('.src.rpm') or
```

Comment:
@ignatenkobrain I am not sure about corret place of operator. Acording to (https://github.com/PyCQA/pycodestyle/issues/197), operator should be at beginning of next line. Please can you provide an evidence? The rest is fine.